src/main/java/edu/uc/javahotkey/JavaHotKeyController.java

```diff
@@ -17,17 +17,16 @@ public String index() {
         return "index";
     }
 
-    @GetMapping("/sampleJsonSchema")
+    @GetMapping("/GetHotKeys{keyMap}")
+    public ResponseEntity fetchAllHotKeys(){return new ResponseEntity(HttpStatus.OK);}
```

Comment:
We need to send a status of OK for when we try and fetch the list of all the Hot Keys when we initially load the project